tsconfig.base.json

```diff
@@ -16,6 +16,6 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES2021"
+    "target": "ES2022"
```

Comment:
Aside: I'm surprised TS 5.4 doesn't support an ES2023 target. I tried that locally and got:

```plaintext

   ✖  nx run ast-spec:build


      ../../tsconfig.base.json(19,15): error TS6046: Argument for '--target' option must be: 'es5', 'es6', 'es2015', 'es2016', 'es2017', 'es2018', 'es2019', 'es2020', 'es2021', 'es2022', 'esnext'.
      Warning: command "yarn build" exited with non-zero status code
```

https://github.com/microsoft/TypeScript/issues/56303 - turns out es2023 === es2022 for target so they omitted it. Huh.

Reply:
Yes, this was rectified in https://github.com/microsoft/TypeScript/pull/58140 and we plan to not skip again.